conf/global.json

```diff
@@ -16,6 +16,10 @@
     ],
     "publisher_locations": [
       "publishers"
+    ],
+    "third_party_libraries": [
+      "pathlib2==2.3.5",
+      "policyuniverse==1.3.2.1"
```

Comment:
the `third_party_libraries` is meant to allow users to inject dependencies that are NOT part of the core `streamalert` package.. meaning, for all of the user-configurable things like `rules`, `publishers`, `matchers`, `schedule_queries`, etc - dependencies _not_ included as part of the core package should be included in `third_party_libraries` for installation at deploy time

does that make sense?

Reply:
cbapi, pymsteams, etc are all dependencies that are needed by the `streamalert` package, so they are hard-coded here.

Reply:
Not right now, but we should revisit these dependencies eventually. Like I don't think `cbapi` should be deployed as part of the base package, since it's only used by the alert processor.

Reply:
disagree :) that is what we _were_ doing and it's garbage. but a different discussion for another time.

Reply:
I only agree with you if we adopt the concept of "subpackages" (https://packaging.python.org/guides/packaging-namespace-packages/), which could be very likely in the next iteration of this